jdaviz/configs/specviz/tests/test_helper.py

```diff
@@ -221,3 +226,21 @@ def test_app_links(specviz_helper, spectrum1d):
     sv = specviz_helper.app.get_viewer('spectrum-viewer')
     assert isinstance(sv.jdaviz_app, Application)
     assert isinstance(sv.jdaviz_helper, Specviz)
+
+
+@pytest.mark.remote_data
+def test_load_spectrum_list_directory(tmpdir):
+    with tempfile.TemporaryDirectory() as tmpdir:
```

Comment:
You don't need `tempfile` when you use the `tmpdir` fixture.

Reply:
Fixed.